docs/postman-setup.md

```diff
@@ -0,0 +1,18 @@
+## Installing Postman on desktop
```

Comment:
@lobsam : Please add information about 
  1. How we generate the schema from graphQL and upload them on postman.
  2. How we document our own custom GraphQL queries and keep it separate from the GraphQL API generated by Hasura.
  3. Link to our postman Team workspace.